src/CLI/Application/App.php

```diff
@@ -24,13 +26,22 @@ public function getVersion(): string
 
     public function getCommands(): array
     {
+        $config = new PHPManagerConfiguration();
+        $fileSystem = new FileSystem();
```

Comment:
```suggestion
        $fileSystem = new Filesystem();
```

(it works because class names, like method names, are case insensitive)